rfcs/2022-07-28-13691-registered-internal-events.md

```diff
@@ -211,7 +211,7 @@ struct RunningSource {
 
 ## Rationale
 
-- This interface for registered internal event handles is notationally equivalent to the existing
+- This interface for registered internal event handles is notionally equivalent to the existing
```

Comment:
This is quite likely wrong, but I decided to ask...

Reply:
I'm actually not sure about either - I think perhaps `functionally` fits, but `notationally` is probably ok as well?

It's attempting to describe how the two interfaces are visually similar/used similarly/written out about the same... 🤔

Reply:
Given the choice of those three, I'd probably lean towards `functionally` even though it technically loses a bit of nuance.

I might even go w/:
```
- This interface for registered internal event handles is similar to the existing
```
or `shaped/structured similarly to...` / `shaped/structured to mirror/follow ...` / `using the _ InternalEvent interface_ as a template/pattern`

If it takes too long to come up w/ something that people likes, I'd of course drop this change. Any improvements overall are better than getting hung up on any one change :)